File: UdidServer/src/horizon/helper/DbHelper.java
package horizon.helper;

import java.sql.PreparedStatement;
import java.sql.ResultSet;
import java.sql.Statement;

public class DbHelper {
    public static void closeQuietly(Statement statement) {
        if (statement != null ) {
            try {
                if(!statement.isClosed()){
                    statement.close();
                }
            } catch (Exception ignore) {
            }
        }
    }

    public static void closeQuietly(PreparedStatement statement) {
        if (statement != null ) {
            try {
                if(!statement.isClosed()){
                    statement.close();
                }
            } catch (Exception ignore) {
            }
        }
    }

    public static void closeQuietly(ResultSet resultSet) {
        if (resultSet != null ) {
            try {
                if(!resultSet.isClosed()){
                    resultSet.close();
                }
            } catch (Exception ignore) {
            }
        }
    }


}

File: UdidServer/src/main/kotlin/tools/helper/DbHelper.kt
package tools.helper

import java.sql.PreparedStatement
import java.sql.ResultSet
import java.sql.Statement

object DbHelper {
    @JvmStatic
    fun closeQuietly(statement: Statement?) {
        if (statement != null) {
            try {
                if (!statement.isClosed) {
                    statement.close()
                }
            } catch (ignore: Exception) {
            }
        }
    }

    fun closeQuietly(statement: PreparedStatement?) {
        if (statement != null) {
            try {
                if (!statement.isClosed) {
                    statement.close()
                }
            } catch (ignore: Exception) {
            }
        }
    }

    fun closeQuietly(resultSet: ResultSet?) {
        if (resultSet != null) {
            try {
                if (!resultSet.isClosed) {
                    resultSet.close()
                }
            } catch (ignore: Exception) {
            }
        }
    }
}
